client/egypt-drug-checker-dashboard/src/components/leader-board/user-card/UserCard.component.tsx

```diff
@@ -0,0 +1,78 @@
+import CardContainer from "@/components/shared/card-container/CardContainer.component";
+import { IuserCard } from "./UserCard.model";
+import { Typography } from "@mui/material";
```

Comment:
same

Reply:
chore(frontend): adjusting name of the typography component to MUtypography